### ISSUE TITLE
Crash when installed with admin rights

### ISSUE THREAD

**Issue by ccbur** [genotrance/px#37]
I'm running into a similar kind of issue like #31. I have px "installed", but after every reboot px crashs with a popup window _Failed to execute script px_.

I have the feeling that the reasons are missing file permissions. When I "install" px, this is done with administrator rights, so the folder where px is installed is writable only for an administrator. But when px is started, this happens without admin rights. The strange thing is that px also crashes when installed without debug setting active, so nothing should be written in the folder. I checked the created px.ini and it's log=0 there.

When I start px.exe manually in an unelevated cmd.exe, it's all fine, no crash. But during boot, always a crash (without a log file, of course).

@genotrance: Any idea how to further debug this?

**Comment by ccbur** [comment]
I was wrong about the file permissions. If I give the user write access to the folder px still crashs! But now I get a debug log:

```
Traceback (most recent call last):
  File "px.py", line 1469, in <module>
  File "px.py", line 1464, in main
  File "px.py", line 1161, in parse_config
  File "px.py", line 1423, in attach_console
  File "site-packages\psutil\__init__.py", line 666, in cmdline
  File "site-packages\psutil\_pswindows.py", line 638, in wrapper
psutil._exceptions.AccessDenied: psutil.AccessDenied (pid=1472)
```

**Comment by ccbur** [comment]
Problem is p.cmdline() fails if the current user has not the access rights to see the cmdline of the parent process(es):

https://github.com/genotrance/px/blob/master/px.py#L1423

But only the executable name is needed, so changing the line to

` if os.path.basename(p.name()).lower() in ["cmd", "cmd.exe", "powershell", "powershell.exe"]:
`
seems to work for me.

**Comment by genotrance** [comment]
This is cool, you've found the real fix for #31. I could never get any logs.

**Comment by ccbur** [comment]
Glad I could help :-)
